src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.iOS.cs

```diff
@@ -84,7 +84,7 @@ private CGRect GetDrawRect(CGRect rect)
 		protected internal override void OnInvalidateMeasure()
 		{
 			base.OnInvalidateMeasure();
-			SetNeedsDisplay();
+			SetNeedsLayout();
```

Comment:
I'm very puzzled as to how this could have any effect. OnInvalidateMeasure is called from UIElement.InvalidateMeasure();  InvalidateMeasure() is already calling SetNeedsLayout().

Reply:
It's because it's not the same SetNeedsLayout() method.
UIElement.InvalidateMeasure(); calls UIView.SetNeedsLayout() and this one calls FrameworkElement.SetNeedsLayout() (which one call UIView.SetNeedsLayout if _inLayoutSubviews is false).
Should I modify the IFrameworkElementHelper.InvalidateMeasure instead of TextBlock.OnInvalidateMeasure ?

Reply:
And do we need SetNeedsDisplay in that case ? SetNeedsLayout will request for a redraw, right ?

Reply:
It's a virtual method, it's the same method.

Reply:
Since it's too random to reproduce, I will just close the PR.